packages/core/src/Modules/Endpoint/Endpoint.jsx

```diff
@@ -99,6 +99,7 @@ const Endpoint = () => {
                         type='submit'
                         is_disabled={
                             !!(
+                                (!touched.server && !touched.app_id && !touched.is_debug_service_worker_enabled) ||
```

Comment:
why do we need this conditional check? cause i see it's the same existing condition at the bottom

Reply:
not sure, might be exidentaly added while working on first implementation of feature flag.
will remove it